systemtest/src/test/java/io/strimzi/systemtest/utils/KafkaVersionUtilsTest.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.utils;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+
+public class KafkaVersionUtilsTest {
+
+    @Test
+    public void parsingTest() throws Exception {
```

Comment:
Not sure I understand why is this test in the systemtests. Could you explain?

Reply:
This was so that the system tests can parse the versions file without having to add additional deps in the system test pom file (the versions code is another module).

Reply:
I chatted with @tombentley about this and we agreed it was easier than importing the whole module for one class, or making a one class common module.

Reply:
So to answer your question, this unit test is to make sure that the kafka-versions parsing code works.

Reply:
Makes sense. Thanks.